.travis.yml

```diff
@@ -14,21 +18,33 @@ services:
 env:
   - DATABASE_URL="postgres://postgres@localhost/jarbas"
 
-install:
-  - psql -U postgres -c 'create database "jarbas";'
+before_install:
+  - |
+    if [ ! -d sysconfcpus/bin ];
```

Comment:
Will be something pretty hard to understand why this is here in the future. Luckily you already documented in the commit.